pkg/cmd/dev/test.go

```diff
@@ -347,6 +347,7 @@ func (d *dev) test(cmd *cobra.Command, commandLine []string) error {
 		args = append(args, "--test_sharding_strategy=disabled")
 	}
 
+	args = append(args, d.getGoTestEnvArgs()...)
```

Comment:
Looks like this reusable fixture stuff is used for benchmarks rather than unit tests (which is the correct behavior as far as I can tell). Can you please add this extra argument to `dev bench` invocations and NOT `dev test` invocations?